themes/finna2/templates/RecordDriver/DefaultRecord/holdings-deduplicated.phtml

```diff
@@ -0,0 +1,33 @@
+<!-- START of: finna - RecordDriver/DefaultRecord/holdings-deduplicated.phtml -->
+<?php
+  $selectID = "{$this->driver->getUniqueID()}-dedup-select";
```

Comment:
ID voi sisältää merkkejä, jotka eivät ole valideja elementin id:ssä. safeId:itä luodaan eri paikoissa poistamalla ei-alfanumeeriset merkit, mikä voi kyllä olla vähän riskikin. Ehkä kuin base64 tuosta voisi olla turvallinen, mutta jos tietueen id:tä ei varsinaisesti tarvitse kaivaa elementin id:stä (johon data-attribuutti olisi parempi), niin ehkä joku `static $counter = 0; ++$counter; $selectID = "dedup-select-$counter";`